### ISSUE TITLE
method missing from compiled code in 2.2.0

### ISSUE THREAD

**Issue by lukekarrys** [lodash/babel-plugin-lodash#37]
I'm seeing and error where the generated code with this plugin is now missing having the method inlined in some cases after updating from `2.1.0` to `2.2.0`.

**source**
```js
import {noop} from 'lodash';
export default (cb) => typeof cb === 'function' ? cb : noop;
```

**compiled**
```js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _noop2 = require('lodash/utility/noop');

var _noop3 = _interopRequireDefault(_noop2);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = function (cb) {
  return typeof cb === 'function' ? cb : ;
};
```

I put together [a gist](https://gist.github.com/lukekarrys/9ed25fa5842036502377#file-readme-md) with a the complete example including my babelrc and the previously compiled code from `2.1.0`.

**Comment by megawac** [comment]
Broke it in https://github.com/lodash/babel-plugin-lodash/commit/20ec9242329b9d0e95b50009dbb286bfd0a7f6cf

**Comment by lukekarrys** [comment]
Thanks @megawac!

**Comment by megawac** [comment]
Fixed in 2.2.1
